doc/nrf/gs_toolchain.rst

```diff
@@ -0,0 +1,19 @@
+.. _gs_toolchain:
+
+Installing the |NCS| from a zip (experimental)
+##############################################
+
+.. warning::
+   Installing from a zip is not officially supported yet.
+   It should work fine for common use cases, but not all scenarios have been tested.
+
+To simplify the |NCS| installation on Windows operating systems, you can download a zip file that contains the full |NCS| toolchain.
+
+The toolchain zip includes the full `GNU Arm Embedded Toolchain`_, the Nordic Edition of |SES|, and all tools that are required to build and program |NCS| applications.
+The |NCS| code base must be downloaded separately as part of the installation process.
+
+To install the |NCS| from a zip, download the toolchain zip from INSERTLINK.
+Then follow the instructions in the :file:`INSTALL.txt` file to set up your environment.
+
+.. note::
```

Comment:
I think this note is not needed if you use the suggestion from line 10. The link will highlight Windows, so it will be noticeable enough. If not, Windows in the title will make this note "deprecated".